CHANGELOG.md

```diff
@@ -2,6 +2,43 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+# Sidecar v11.0.0 Changelog
+
+## [11.0.0](https://github.com/paritytech/substrate-api-sidecar/compare/v10.0.0..v11.0.0) (2021-10-20)
+
+**Upgrade priority**: Medium (For users leveraging `/accounts/{accountId}/vesting-info`)
+
+### ⚠ BREAKING CHANGES ⚠
+
+- fix!: adjust the vesting-info response to be in line with substrate ([#717](https://github.com/paritytech/substrate-api-sidecar/pull/717)) ([8b9866d](https://github.com/paritytech/substrate-api-sidecar/commit/8b9866d02ab4aa2ab22a19159a8c90a8ddfc9a1b))
```

Comment:
Right, substrate now returns an array because an account could have multiple vestich schedules. I suggest "Support multiple vesting schedules. `vesting-info` now returns an array."

@TarikGul your call!